crates/services/txpool_v2/src/shared_state.rs

```diff
@@ -178,4 +180,8 @@ impl SharedState {
             .write_pool_requests_sender
             .try_send(WritePoolRequest::RemoveCoinDependents { transactions });
     }
+
+    pub fn get_latest_stats(&self) -> TxPoolStats {
```

Comment:
bikeshed: `latest_stats` would be a cleaner name

Reply:
Done